server/converters/cxgtool.py

```diff
@@ -398,44 +408,87 @@ def save_embeddings(container, adata, ctx):
             log(1, f"\t\t...{name} embedding created")
 
 
-def create_X(X_name, shape):
+def create_X(X_name, shape, is_sparse):
     """
-    Dense, always.  Future task: explore if sparse encoding is worth the trouble
-    below a sparsity threshold.
-
     The X matrix is access in both row and column oriented patterns, depending on the
     particular operation.  Because of the data type, default compression works best.
-    The tile size (50, 100) and global layout (row/col) was choosen empirically, by benchmarking
+    The tile size (50, 100) and global layout (row/col) was chosen empirically, by benchmarking
```

Comment:
please update the comment to include information about the sparse tile sizes (I assume they were also empirically discovered based on perf?)

Reply:
They were chosen based on a relatively small amount of benchmarking.   I intend to fine too this as a follow-on update.  I'll update the comment though